frontend/components/Meetups/MeetupsTable.vue

```diff
@@ -1,52 +1,117 @@
 <script setup>
+//work in progress
+//filteredMeetups need to be improve
```

Comment:
If this is not ready for review, you can always open "draft pull request". If it's ready, then I would expect to have no comments like this.

Reply:
Right, I forgot to remove it